### ISSUE TITLE
Support NickServ AJOIN

### ISSUE THREAD

**Issue by dwfreed** [solanum-ircd/solanum#121]
This is implemented in oftc-hybrid as a fake join command (literally just builds the parameters up and calls m_join).  As such, anything that would prevent the user from joining still applies (and it outright doesn't work with +k channels because there's no way to provide the channel key).

Atheme's ns_ajoin contrib module uses s2s syntax that matches oftc-hybrid already, so the same could be used here as well; alternately, adjusting ns_ajoin to use the same syntax as used with elemental-ircd (ENCAP SVSJOIN) would work as well.

**Comment by Neo-Desktop** [comment]
I was looking into this functionality for a personal Solanum/TS6 services project and it appears that this feature isn't fleshed out yet

For anyone else Googling trying to figure out how to force-join a remote user to a channel:
Emitting a JOIN or SJOIN for a user managed under a remote SID will not work, therefore a specific s2s SAJOIN or SVSJOIN is required

**Comment by dwfreed** [comment]
Yes, that's why this is still an open issue.  s2s `JOIN` and `SJOIN` are only for a client's server to report a client joining a channel, or report the clients from that server in a particular channel.  No work has been done to implement this feature, or there'd be linked commits or PRs.
